project/giter8.sbt

```diff
@@ -1 +1,2 @@
 addSbtPlugin("org.foundweekends.giter8" %% "sbt-giter8" % "0.11.0-M4")
+libraryDependencies += { "org.scala-sbt" %% "scripted-plugin" % sbtVersion.value }
```

Comment:
This forces to use the latest scripted-plugin, which is empty. - https://github.com/sbt/sbt/blob/v1.2.0-RC1/scripted/plugin/src/main/scala/sbt/ScriptedPlugin.scala